contracts/optimistic-ethereum/libraries/utils/Lib_TimeboundRingBuffer.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.7.0;
+
+/* Logging */
+import { console } from "@nomiclabs/buidler/console.sol";
```

Comment:
This should be removed before this is merged right?

Reply:
Oh yes it should be! Good catch